README.md

```diff
@@ -1,2 +1,21 @@
-# landscape-analysis
-Enabling landscape analysis to better understand existing efforts or solutions.
+# Landscape Analysis
+
+A repository for analyzing various landscape efforts or solutions for the [Way Lab](https://www.waysciencelab.com/).
```

Comment:
Thanks! Yes, I feel this will be helpful for multiple landscape analyses which may follow a similar structure. While it's not yet clear, I think there may be utility in common functions or modules which could be implemented for multiple efforts.